core/src/types/metadata.rs

```diff
@@ -75,7 +75,7 @@ impl Metadata {
     }
 
     /// Get the bit from metadata.
-    pub(crate) fn bit(&self) -> FlagSet<Metakey> {
+    pub fn metakey(&self) -> FlagSet<Metakey> {
```

Comment:
Please polish the API docs. Maybe we should also rename bit to metakey. Besides, maybe we should also update with_bit?

Reply:
Ok, should we update `contains_bit` together?

Reply:
Yep, let's do this.